src/scrabble.js

```diff
@@ -1,6 +1,49 @@
 class Scrabble {
-  // Write your implementation here
+  constructor (word) {
+    this.word = word
+    this.onePoint = ['a', 'e', 'i', 'o', 'u', 'l', 'n', 'r', 's', 't']
+    this.twoPoints = ['d', 'g']
+    this.threePoints = ['b', 'c', 'm', 'p']
+    this.fourPoints = ['f', 'h', 'v', 'w', 'y']
+    this.fivePoints = ['k']
+    this.eightPoints = ['j', 'x']
+    this.tenPoints = ['q', 'z']
+  }
 
+  score () {
+    let finalScore = 0
+    let subScore = 0
+    let multiply = 1
```

Comment:
nice use of a multiple to adjust the scoring system, I think we will implement this a lot when making minigames